wandb/sdk/wandb_settings.py

```diff
@@ -127,7 +128,7 @@ def _build_inverse_map(prefix: str, d: Dict[str, Optional[str]]) -> Dict[str, st
 
 
 def _error_choices(value: str, choices: Set[str]) -> str:
-    return "{} not in {}".format(value, ",".join(list(choices)))
+    return "{} not in [{}]".format(value, ", ".join(list(choices)))
```

Comment:
i notice you like spaces after commas here but not at line 595 :)
i agree, just stood out to me. :)
i guess because here it is presented as a list.

Reply:
yeah... :)